packages/codeeditor/style/index.css

```diff
@@ -30,23 +30,51 @@
 }
 
 
-.jp-JSONEditor-buttons {
-  font-family: FontAwesome;
+.jp-JSONEditor-header {
+  display: flex;
   flex: 1 0 auto;
   min-height: 13px;
-  padding-bottom: 6px;
+  padding: 12px;
+}
+
+
+.jp-JSONEditor-header label {
+  flex: 0 0 auto;
 }
 
 
 .jp-JSONEditor-commitButton::before {
+  font-family: FontAwesome;
   content: '\f00c'; /* check */
-  float: right;
-  margin-right: 20px;
+  padding-left: 4px;
 }
 
 
 .jp-JSONEditor-revertButton::before {
+  font-family: FontAwesome;
   content: '\f0e2'; /* undo */
-  float: right;
-  margin-right: 4px;
+}
+
+
+.jp-JSONEditor-collapser {
+  flex: 1 1 auto;
+}
+
+
+.jp-JSONEditor-collapser.jp-mod-collapse-enabled::before {
+  padding-left: 6px;
+  font-family: FontAwesome;
+  content: '\f0d8'; /* caret-up */
+}
+
+
+.jp-JSONEditor-collapser.jp-mod-collapse-enabled.jp-mod-collapsed::before {
+  padding-left: 6px;
+  font-family: FontAwesome;
+  content: '\f0d7'; /* caret-down */
```

Comment:
We are lacking MD icons for caret-up and and accept.  I had previously used a save icon but that was confusing because it seemed like you were saving the document.